src/test/java/net/openhft/chronicle/bytes/BytesPrimitiveParameterTest.java

```diff
@@ -87,7 +88,7 @@ Stream<DynamicTest> negativeParameters() {
                     if (isReadWrite(args)) {
                         // Make sure nothing changes in the contents of the Bytes object
                         if (!SILLY_NAME.equals(bytes.toString())) {
-                            System.out.println(bytes.getClass().getSimpleName()+", b: "+ Arrays.toString(bytes.toString().getBytes(StandardCharsets.UTF_8)));
+                            System.out.println(bytes.getClass().getSimpleName() + ", b: " + Arrays.toString(bytes.toString().getBytes(StandardCharsets.UTF_8)));
```

Comment:
could static import Arrays

Reply:
Having a static import for toString(...) could be confusing.